### DISCUSSION TITLE
Connection configuration lost between containers instances

### DISCUSSION THREAD

**Discussion by phergoualch** [open-webui/open-webui#4215] [Q&A]
I'm encountering an issue while running OpenWebUI and LiteLLM on AWS ECS using Fargate Spot instances. Since the containers are ephemeral and can be replaced, I'm losing the OpenAI API URL, API key (for LiteLLM), and default model configurations whenever a new container instance is launched.

I have an external Postgres database set up, which is working correctly. However, the connection configuration is not being persisted across container instances.

Why are these settings not stored in the Postgres database along with the users ? 
Is there a specific file or directory that I should make persistent to maintain these configurations between container instances ?

Thanks

**Comment by justinh-rahb** [comment]
You should use environment variables for your configuration in this case:
https://docs.openwebui.com/getting-started/env-configuration
